<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="HPENVY" name="Marketing_Project.New_Scenarios" tests="12" failures="0" timestamp="14 Jul 2021 12:17:51 GMT" time="373.428" errors="1">
  <testcase name="Invoice_Setup" time="18.202" classname="Marketing_Project.New_Scenarios"/>
  <testcase name="Email_Template_setup" time="21.495" classname="Marketing_Project.New_Scenarios"/>
  <testcase name="Add_Customer_and_Service" time="84.981" classname="Marketing_Project.New_Scenarios"/>
  <testcase name="Add_Meter" time="23.117" classname="Marketing_Project.New_Scenarios"/>
  <testcase name="Add_Meter_Register" time="14.138" classname="Marketing_Project.New_Scenarios"/>
  <testcase name="Add_Meter_Reads" time="46.399" classname="Marketing_Project.New_Scenarios"/>
  <testcase name="Add_BillRun_Cycle" time="26.434" classname="Marketing_Project.New_Scenarios"/>
  <testcase name="Bill_Run" time="26.478" classname="Marketing_Project.New_Scenarios"/>
  <testcase name="Update_Statement_Dates" time="15.875" classname="Marketing_Project.New_Scenarios"/>
  <testcase name="Download_PDF" time="5.363" classname="Marketing_Project.New_Scenarios"/>
  <testcase name="Email" time="62.542" classname="Marketing_Project.New_Scenarios">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Re_Useable.Assertion_Test.Assertion_Send_Email_Results_Popup(Assertion_Test.java:520)
at Marketing_Project.New_Scenarios.Email(New_Scenarios.java:763)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Email -->
  <testcase name="Bill_Run_With_Uncommitted_Statement" time="28.404" classname="Marketing_Project.New_Scenarios"/>
</testsuite> <!-- Marketing_Project.New_Scenarios -->
